### ISSUE TITLE
ID 256132 : [Programmatic Access - Simon - Home]: Incorrect aria-label is defined for "Grafana Play Home" link on 'Home' page.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#65801]
### **User Experience:** 
Visually impaired people and cognitive disabled users who depend on screen readers will get disoriented and will face difficulty in understanding the control if proper name is not defined for the controls. 

_Note: User credentials should NOT be included in the bug._

### **Repro Steps**:
Open #Url: [play.grafana.org](https://play.grafana.org/). Sign in with valid credentials. "Home" page appears. Navigate to 'Grafana Play Home' in the header. Turn on narrator. Verify the issue.

**Actual Result**: 
Issue: Incorrect `aria-label` is defined for "Grafana Play Home" link as "Search dashboard by name" on 'Home' page.

**Observation**: 
Screen reader is announcing "Search dashboard by name" link when focus moves on it.

**Expected Result:** 
Correct `aria-label `should be defined as "Grafana Play Home" for "Grafana Play Home" link on 'Home' page.

### **Severity:** 
`SEV3`

### **Area:**
`Top Toolbar`

**Comment by grafanabot** [comment]
@eledobleefe please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one label that best describes the product or feature area. Please also add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one label that best describes the product or feature that you are discussing (e.g. `area/alerting`, `datasource/loki`, `type/docs`, `type/bug`, etc). [Our issue triage](https://github.com/grafana/grafana/blob/main/ISSUE_TRIAGE.md#3-categorizing-an-issue) doc also provides additional guidance on labeling. :rocket:

 Thank you! :heart:

**Comment by eledobleefe** [comment]
Fixed in the new nav.
